### PR TITLE
fix(lastVisited): handles last visited to get existing route

### PR DESCRIPTION
**Pull request by nickjjang**
AP-372

### CODE REVIEW

**Review by Jekrimo — APPROVED** [review]

File: pages/chat/direct/_address.vue [hunk 1]
```diff
@@ -25,9 +25,19 @@ export default Vue.extend({
     // This information can be useful for users to help us find and report bugs.
     ConsoleWarning(this.$config.clientVersion, this.$store.state)
     const address = this.$route.params.address
+    const { all: friends } = this.$store.state.friends
     if (address) {
-      this.$store.dispatch('textile/fetchMessages', { address })
+      const friend = friends.find((fr) => fr.address === address)
```
Comment (RetroPronghorn) [inline pages/chat/direct/_address.vue:30]: Can we add this to houndDog?

**Review by RetroPronghorn — CHANGES_REQUESTED** [review]

File: pages/chat/direct/_address.vue (hunk 1 above)
Comment (nickjjang) [reply pages/chat/direct/_address.vue]: Added it

**Review by WanderingHogan — APPROVED** [review]

**Review by stavares843 — CHANGES_REQUESTED** [review]
still getting the same error as the one on the ticket - AP-372

Captura de ecrã 2022-01-17, às 20 45 24

how to reproduce
- click on the satellite icon on the top left corner
Captura de ecrã 2022-01-17, às 20 44 30
- returns the error above

**Comment by stavares843** [comment]
https://user-images.githubusercontent.com/29093946/150398956-5701d5e1-784b-4383-b77c-1ff463364f2c.mov

retested and im getting the above error

**Comment by nickjjang** [comment]
Hi @stavares843 I've fixed it.

**Comment by stavares843** [comment]
thanks! 🔨 checking

**Comment by stavares843** [comment]
fixed ✅

**Review by stavares843 — APPROVED** [review]

**Pull request merged**